app/src/main/java/ar/com/wolox/android/cookbook/graphQl/OrdersPresenter.kt

```diff
@@ -0,0 +1,35 @@
+package ar.com.wolox.android.cookbook.graphQl
+
+import android.util.Log
+import ar.com.wolox.android.cookbook.model.ModelOrder
+import ar.com.wolox.android.cookbook.netowork.GraphQlService
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import com.apollographql.apollo.ApolloCall
+import com.apollographql.apollo.api.Response
+import com.apollographql.apollo.exception.ApolloException
+import com.wolox.queries.GetOrdersQuery
+import javax.inject.Inject
+
+class OrdersPresenter @Inject constructor() : BasePresenter<OrdersView>() {
+    private var orderList: ArrayList<ModelOrder>? = null
+
+    fun init() {
```

Comment:
you can avoid this ommit by calling onViewAttached. This will run before view.init() so make sense to your app and you avoid using the view to give "orders" to the presenter.

Reply:
good to know!